src/app/home/header/header.component.html

```diff
@@ -1,2 +1,5 @@
-<p>header works!</p>
-<h1><strong>Placeholder for Header</strong></h1>
+<mat-toolbar color="primary">
+  <span><a routerLink="/">vitaL</a></span>
+  <span class="spacer"></span>
+  <ngx-auth-firebaseui-avatar></ngx-auth-firebaseui-avatar>
```

Comment:
If this is just a placeholder for demo, we can add a task for the sign out implementation in backlog.

Reply:
No this is not a placeholder. It works. Auth guards implementation can make it better.

Reply:
Task in backlog to improve on this VIT-100

Reply:
Okay - I am not so clear on it.  It is okay for now, we can work on it when we do auth guard.